plugins/CoreAdminHome/vue/src/JsTrackingCodeGenerator/JsTrackingCodeAdvanceOptions.vue

```diff
@@ -0,0 +1,460 @@
+<template>
+  <div class="trackingCodeAdvanceOptions">
+    <div class="advance-option">
+    <span>
+      <a href="javascript:;"
```

Comment:
This probably shouldn't be an `<a>` but rather a `<button>` unless we have that as a common patter already established.

Reply:
I think we have it as anchor tag